packages/block-library/src/file/edit.js

```diff
@@ -33,6 +34,10 @@ import { store as noticesStore } from '@wordpress/notices';
  * Internal dependencies
  */
 import FileBlockInspector from './inspector';
+import { browserSupportsPdfs } from './utils';
+
+export const MIN_PREVIEW_HEIGHT = 200;
+export const MAX_PREVIEW_HEIGHT = 2000;
```

Comment:
Also this seems huge to me 😄. Was it like this before and missed it in my first round?

Reply:
It's always been like that: I figure if people want to set their embed to be huge, they can.